gulp-core-build-karma/karma.config.js

```diff
@@ -4,11 +4,14 @@
 // Generated on Thu Oct 08 2015 18:13:05 GMT-0700 (PDT)
```

Comment:
The real problem here is the odd inversion of control that happens. TBH the correct solution would be editing gulp-karma to allow passing the tsconfig directly in. This would allow us to do something similar to what we do with webpack, where we have a task which generates the configuration. The problem with the way KarmaTask.ts is written is that it expects you to provide your own karma config (this file is the sample file it will drop in your project). So technically this file only affects new projects using web-library-build.